1-exercises/D-for-in-loops/exercise-1.js

```diff
@@ -13,7 +13,11 @@ let salaries = {
 };
 
 // WRITE BELOW THIS LINE
-
+for(const el in salaries){
```

Comment:
I'd probably opt for a more meaningful name in a situation like this (e.g. `salary`)